packages/rpc-graphql/src/__tests__/account-test.ts

```diff
@@ -845,53 +828,44 @@ describe('account', () => {
-                    data: {
-                        account: {
-                            address: 'AyGCwnwxQMCqaU4ixReHt8h5W4dwmxU7eM3BEQBdWVca',
-                            ownerProgram: {
-                                address: 'TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA',
-                                ownerProgram: {
-                                    address: 'BPFLoader2111111111111111111111111111111111',
-                                },
-                            },
-                        },
-                    },
-                });
+                await rpcGraphQL.query(source);
                 expect(fetchMock).toHaveBeenCalledTimes(2);
             });
         });
     });
     describe('cache tests', () => {
         it('coalesces multiple requests for the same account into one', async () => {
-            expect.assertions(2);
+            expect.assertions(1);
             const source = `
                 query testQuery {
-                    account(address: "Gh9ZwEmdLJ8DscKNTkTqPbNwLNNBjuSzaG9Vp2KGtKJr") {
-                        address
-                        ... on MintAccount {
-                            mintAuthority {
-                                address
-                                lamports
-                            }
-                        }
+                    account1: account(address: "Gh9ZwEmdLJ8DscKNTkTqPbNwLNNBjuSzaG9Vp2KGtKJr") {
+                        lamports
+                    }
+                    account2: account(address: "Gh9ZwEmdLJ8DscKNTkTqPbNwLNNBjuSzaG9Vp2KGtKJr") {
+                        lamports
+                    }
+                    account3: account(address: "Gh9ZwEmdLJ8DscKNTkTqPbNwLNNBjuSzaG9Vp2KGtKJr") {
+                        lamports
                     }
                 }
             `;
-            const result = await rpcGraphQL.query(source);
-            expect(result).toMatchObject({
-                data: {
-                    account: {
-                        address: 'Gh9ZwEmdLJ8DscKNTkTqPbNwLNNBjuSzaG9Vp2KGtKJr',
-                        mintAuthority: {
-                            address: 'Gh9ZwEmdLJ8DscKNTkTqPbNwLNNBjuSzaG9Vp2KGtKJr',
-                            lamports: expect.any(BigInt),
-                        },
-                    },
-                },
-            });
+            await rpcGraphQL.query(source);
             expect(fetchMock).toHaveBeenCalledTimes(1);
         });
+        it('cache resets on new tick', async () => {
+            expect.assertions(1);
+            await jest.runAllTimersAsync();
+            const source = /* GraphQL */ `
+                query testQuery {
+                    account(address: "Gh9ZwEmdLJ8DscKNTkTqPbNwLNNBjuSzaG9Vp2KGtKJr") {
+                        lamports
+                    }
+                }
+            `;
+            // Call the query twice
+            await rpcGraphQL.query(source);
+            await rpcGraphQL.query(source);
```

Comment:
I've got to be missing something here, because if I do that, the `fetchMock` is called 0 times. Maybe I need to do some setup or something. I'll check it out.